envisage/tests/support.py

```diff
@@ -42,8 +42,10 @@
     import PySide6
 except ImportError:
     pyside6_available = False
+    pyside6_version = None
 else:
     pyside6_available = True
+    pyside6_version = PySide6.__version_info__
```

Comment:
Should we use importlib.metatdata here?

Reply:
I considered it, but we don't currently have a dependency on `importlib.metadata`, and I was reluctant to add one just for this, particularly given that this is code that will be going away as soon as Pyface is released.